routes/truckRoutes.js

```diff
@@ -1,3 +1,67 @@
-// connect to mongodb in this module as this is where you'll be making creat/read/delete calls to your database
-// use 'mongodb://localhost/foodTruckAPI' for your mongoose connection string
-// remember this is a Node module
+var express = require('express');
+var mongoose = require('mongoose');
+var Truck = app.route('../models/truckModel');

```

Comment:
When you define Truck here, you want to set it to the result of a require call:

`var Truck = require('../models/truckModel');`

You don't need to use app.route anywhere in your application because you'll be using a router instance which you create below on line 5.